mlflow/deployments/cli.py

```diff
@@ -0,0 +1,154 @@
+import click
+from mlflow.utils import cli_args
+from mlflow.deployments import interface
+
+
+def _user_args_to_dict(user_list):
+    # Similar function in mlflow.cli is throwing exception on import
+    user_dict = {}
+    for s in user_list:
+        try:
+            name, value = s.split('=')
+        except ValueError:
+            # not enough values to unpack
+            raise click.BadOptionUsage("config", "Config options must be a pair and should be"
+                                                 "provided as ``--C key=value`` or "
+                                                 "``--config key=value``")
+        if name in user_dict:
+            raise click.ClickException("Repeated parameter: '{}'".format(name))
+        user_dict[name] = value
+    return user_dict
+
+
+target_uri = click.option("--target-uri", "t_uri", required=True,
+                          help="Deployment target URI. Check the documentation/help for each "
```

Comment:
It'd be cool to include the name of all currently-installed plugins in the help string as a bulleted list, e.g something like:

```
help="""
Deployment target URI, e.g. 'sagemaker'.  Run `mlflow deployments help -t <target-name>`
for more details on the supported URI format and config options for a given target.

Currently-supported targets (see other supported deployment targets and instructions for installing support in  https://mlflow.org/docs/latest/plugins.html#community-plugins):
{supported_targets}
""".format(supported_targets="\n".join(["* %s" % target for target in list_of_targets_from_plugin_registry)]))
```

Reply:
Currently, supported or currently installed in the user's machine? Because we don't have an automated way of getting currently supported plugins unless we hardcode it there